### PR TITLE
Add geoutils

### PR DESCRIPTION
**Pull request by erikmannerfelt**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/geoutils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+source:
+        url: "{{ github_url }}/archive/refs/tags/{{ version }}.tar.gz"
+        sha256: "7efde1a137978754d847370b2fbd7b458bd404cad3c265f4fc9ba4fba8891cfd"
+
+
+build:
+        number: 0
+        script: python -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+        preserve_egg_dir: True
+        noarch: generic
```
Comment (wolfv) [inline recipes/geoutils/meta.yaml:18]: noarch: python

File: recipes/geoutils/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  version: "{{ version }}"
+source:
+        url: "{{ github_url }}/archive/refs/tags/{{ version }}.tar.gz"
+        sha256: "7efde1a137978754d847370b2fbd7b458bd404cad3c265f4fc9ba4fba8891cfd"
+
+
+build:
+        number: 0
+        script: python -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+        preserve_egg_dir: True
```
Comment (wolfv) [inline recipes/geoutils/meta.yaml:17]: is this on purpose?

File: recipes/geoutils/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+source:
+        url: "{{ github_url }}/archive/refs/tags/{{ version }}.tar.gz"
+        sha256: "7efde1a137978754d847370b2fbd7b458bd404cad3c265f4fc9ba4fba8891cfd"
+
+
+build:
+        number: 0
+        script: python -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
```
Comment (wolfv) [inline recipes/geoutils/meta.yaml:16]: better to use `{{ PYTHON }}` instead of `python` to make sure you're using the right python

**Review by wolfv — CHANGES_REQUESTED** [review]
some changes requested :)

File: recipes/geoutils/meta.yaml (hunk 3 above)
Comment (erikmannerfelt) [reply recipes/geoutils/meta.yaml]: Fixed in 40a7623

File: recipes/geoutils/meta.yaml (hunk 2 above)
Comment (erikmannerfelt) [reply recipes/geoutils/meta.yaml]: Nope, sorry! Removed in 40a7623

File: recipes/geoutils/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
+{% set name = "geoutils" %}
+{% set version = "v0.0.1" %}
+
+{% set github_url = "https://github.com/GlacioHack/GeoUtils" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+source:
+        url: "{{ github_url }}/archive/refs/tags/{{ version }}.tar.gz"
```
[the remark this answers is not in the document]
Comment (erikmannerfelt) [reply recipes/geoutils/meta.yaml]: I have no idea how that happened. Sorry about that! Made all indentations with 2 spaces in 40a7623.

**Pull request merged**